### DISCUSSION TITLE
How can I translate MOTIS?

### DISCUSSION THREAD

**Discussion by kalon33** [motis-project/motis#257] [Q&A] [answered]
Is it possible to translate MOTIS in another language?

**Comment by felixguendling** [comment, marked as answer]
I assume you are referring to the web user interface?

There are already localizations for German, English and Polish:
https://github.com/motis-project/motis/tree/master/ui/web/src/Localization

**Reply by kalon33** [reply]
Are there similar files for the Android app? I will take some time to translate it like the web user interface if it is possible.

Thanks!

**Reply by felixguendling** [reply]
Strings are here: https://github.com/motis-project/motis/blob/master/ui/android/app/src/main/res/values/strings.xml
